iml-agent/src/network_interfaces.rs

```diff
@@ -48,9 +49,19 @@ pub async fn get_interfaces() -> Result<Vec<NetworkInterface>, ImlAgentError> {
 }
 
 pub async fn get_lnet_data() -> Result<LNet, ImlAgentError> {
-    let output = get_lnet_data_cmd().checked_output().await?;
+    let r = get_lnet_data_cmd().checked_output().await;
 
-    let lnet_data = std::str::from_utf8(&output.stdout)?.trim();
+    let x = match r {
+        Ok(x) => Ok(x.stdout),
+        Err(CmdError::Io(ref err)) if err.kind() == io::ErrorKind::NotFound => {
+            tracing::debug!("lnetctl was not found. Will not send net data");
```

Comment:
Maybe a warning?

Reply:
This is pretty common; it's going to happen until lustre is installed on the server.